balancer/catabalancer/sysstats.go

```diff
@@ -63,3 +95,31 @@ func GetSystemUsage() (SystemUsage, error) {
 
 	return systemUsage, nil
 }
+
+func GetBandwidthUsage() (float64, error) {
+	config := VnstatConfig{
+		Interface: "eth0",
+		Args:      []string{"-tr", "2.5"},
+	}
+	args := append([]string{"--json", "--iface", config.Interface}, config.Args...)
+
+	cmd := exec.Command("vnstat", args...)
+	cmd.Env = os.Environ()
+
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return -1, err
+	}
+
+	var data BandwidthData
+	err = json.Unmarshal([]byte(output), &data)
+	if err != nil {
+		return -1, err
+	}
+
+	totalBW := float64(data.Rx.Bytes) + float64(data.Tx.Bytes)
+	// TODO: is our NICs 20Gbps?? 
+	totalBWPercent := (totalBW / (20 * 1e9)) * 100
```

Comment:
hmmmm maybe not? 
```
root@prod-catalyst-0:/# cat /sys/class/net/eth0/speed
10000
```

Reply:
Yeah, 10 sounds right but it varies by machine

Reply:
You can confirm via the Datapacket website though